carta/cpp/plugins/WcsPlotter/plugin.json

```diff
@@ -7,5 +7,5 @@
         "Plots WCS grids using starlink AST lib"
     ],
     "about"      : "Written by Pavol Federl",
-    "depends"    : [ "casaCore-2.0.1", "CasaImageLoader"]
+    "depends"    : [ "casaCore-2.0.1", "CasaImageLoader", "cartaview"]
```

Comment:
This plugin does not need to depend directly on cartaview as it already depends on casaimageloader, which in turn depends on cartaview.

It also does not need to directly depend on casacore-2.0.1 for the same reason (I know, I introduced this one myself:)

Same goes for any other plugin...